src/main/java/org/opentripplanner/routing/edgetype/TripPattern.java

```diff
@@ -91,6 +92,9 @@
      */
     public final Timetable scheduledTimetable = new Timetable(this);
 
+    /** Map of trip IDs to current vehicle positions. Will be an empty array if none are found. */
```

Comment:
The map is an empty array if none are found? Or the vehicle positions will be an empty array. I'm assuming the latter, but the comment is kind of confusing at first.

Reply:
Straggler from back before this was a map. Good catch thank you! Refactored in 94fe2b070ea0ab38e10e88f37b271586688d48bd